docs/docsite/rst/dev_guide/testing/sanity/index.rst.new

```diff
@@ -0,0 +1,46 @@
+Sanity Tests
```

Comment:
Added accidently?

Reply:
grrr... still not sure why that file keeps showing up... removing now, thanks!